packages/vue/src/components/molecules/SfCheckbox/SfCheckbox.stories.js

```diff
@@ -242,7 +242,6 @@ const Template = (args, { argTypes }) => ({
     style="--checkbox-container-width: 15rem;"
     :name="name"      
     :label="label"
-    :hintMessage="hintMessage"
```

Comment:
I think we shouldn't remove this prop here. There is a default value of `hintMessage` passed here and it is an empty string. So there is only a need for passing 'hintMessage' value like `Required`.

Reply:
done:)